client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -455,9 +465,44 @@ def _create_thumbnail_oiio(
 
         return True
 
+    def _create_thumbnail_oiio(self, src_path, dst_path):
+        self.log.debug(f"Extracting thumbnail with OIIO: {dst_path}")
+
+        try:
+            resolution_arg = self._get_resolution_arg("oiiotool", src_path)
+        except RuntimeError:
+            return False
+
+        input_info = get_oiio_info_for_input(src_path, logger=self.log)
+        input_arg, channels_arg = get_oiio_input_and_channel_args(input_info)
+        oiio_cmd = get_oiio_tool_args(
+            "oiiotool",
+            input_arg, src_path,
+            # Tell oiiotool which channels should be put to top stack
+            #   (and output)
+            "--ch", channels_arg,
+            # Use first subimage
+            "--subimage", "0"
+        )
+        oiio_cmd.extend(resolution_arg)
+        oiio_cmd.extend(("-o", dst_path))
+        self.log.debug("Running: {}".format(" ".join(oiio_cmd)))
+        try:
+            run_subprocess(oiio_cmd, logger=self.log)
+            return True
+        except Exception:
+            self.log.warning(
+                "Failed to create thumbnail using oiiotool",
+                exc_info=True
+            )
+            return False
+
     def _create_thumbnail_ffmpeg(self, src_path, dst_path):
-        self.log.debug("Extracting thumbnail with FFMPEG: {}".format(dst_path))
-        resolution_arg = self._get_resolution_arg("ffmpeg", src_path)
+        try:
+            resolution_arg = self._get_resolution_arg("ffmpeg", src_path)
+        except RuntimeError:
+            return False
```

Comment:
Do we lack a logged message that FFMPEG conversion failed? Just so that when reading the logs it's clear that both OIIO and FFMPEG may have failed or not?